subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/UITestConfig.kt

```diff
@@ -40,6 +40,13 @@ object UITestConfig {
 
     var clicksType: ClickType = defaultClicksType
 
+    const val defaultVisualizeClicks = true
```

Comment:
Why do we expose this value?

Reply:
Have a look at `subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/PerformanceTestRule.kt`. This rule is able to restore default value easily and without saving original value on its side.

Reply:
It can clear a custom value. If someone is already disabled it, we will enable it.

I think we should save and restore to the current value

Reply:
You're right. I will fix it. Thank you for spotting the problem.